src/app/js/views/connection.js

```diff
@@ -2,23 +2,15 @@ import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 import Peer from './peer'
 
-export
-default class Connection extends Component {
-  static displayName = 'Connection';
-  static propTypes = {
-    ipfs: PropTypes.object,
-    location: PropTypes.object,
-    bytesRead: PropTypes.number,
-    bytesWritten: PropTypes.number,
-    peer: PropTypes.object,
-    addr: PropTypes.object
-  };
-
-  state = {
-    open: false
-  };
+class Connection extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      open: false
+    }
+  }
 
-  _handleClick = () => {
+  _handleClick () {
```

Comment:
right, almost missed these react'ism.

However, sintaxwise, from what I can see in other examples, it is preferable to bind things in the constructor. Wasn't this working only due thanks to some magic transpilation step in the same way that https://github.com/ipfs/webui/pull/595#discussion_r123908171 was?